test/built-ins/NativeErrors/toString.js

```diff
@@ -0,0 +1,23 @@
+// Copyright 2019 Ecma International. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: NativeError prototypes do not have an own toString method.
+info: |
+  19.5.6.3 Properties of the NativeError Prototype Objects
+
+  Each NativeError prototype object:
+    ...
+    has a [[Prototype]] internal slot whose value is %Error.prototype%.
+---*/
+
+var nativeErrors = [
+  EvalError, RangeError, ReferenceError, SyntaxError, TypeError, URIError
```

Comment:
this test should be split and placed within each respective constructor folder for consistency within the project. The esid should connect to each one's section.

- `test/built-ins/NativeErrors/EvalError/toString.js`
- `test/built-ins/NativeErrors/RangeError/toString.js`
- `test/built-ins/NativeErrors/ReferenceError/toString.js`
- `test/built-ins/NativeErrors/SyntaxError/toString.js`
- `test/built-ins/NativeErrors/TypeError/toString.js`
- `test/built-ins/NativeErrors/URIError/toString.js`

This is already present for AggregateError in `test/built-ins/AggregateError/toString.js`.

Note that it's also nice to test the instance of each constructor will also have toString from the Error.prototype.

Reply:
This seems like a lot of duplication. Wouldn't we only want to separate things that are different between them, like `.prototype.name`?